distribution/changelog.txt

```diff
@@ -8,6 +8,7 @@
 - Feature: [#22213] [Plugin] Allow plugins to focus on textboxes in custom windows.
 - Feature: [#22272] [Plugin] Expose ride vehicle’s current track type via car trackLocation.
 - Feature: [#22301] Loading save games or scenarios now indicates loading progress.
+- Feature: [#22472] New title sequences (https://github.com/OpenRCT2/title-sequences/releases/tag/v0.4.13).
```

Comment:
Having a URL in the changelog like this is a bit odd, I think. Maybe we could mention the PR# on the title sequence repo instead?

Reply:
Or just leave it out - it already points to this PR after all, where people can click through if they want to know more.

Reply:
Hmm, last year you asked for a way to credit the creators more easily, and linking to the release page where their names will be listed was whta I chose to do. Linking to the PR or to this one is still at least one more click away form their credits.
https://github.com/OpenRCT2/OpenRCT2/pull/20680#issuecomment-1664254930

Reply:
Maybe put the names in the credits page?

Reply:
We could also put their names into contributors.md.

Nevertheless, if we must do it this way, we should at least clarify why the URL is there.

```suggestion
- Feature: [#22472] New title sequences (see https://github.com/OpenRCT2/title-sequences/releases/tag/v0.4.13 for credits).
```